e2e/test/scenarios/dashboard-filters-2/dashboard-filters-query-stages.cy.spec.ts

```diff
@@ -1793,42 +1709,6 @@ describe("scenarios > dashboard > filters > query stages", () => {
             values: ["1,077"],
           });
         });
-
-        // TODO: https://github.com/metabase/metabase/issues/48339
-        // TODO: https://github.com/metabase/metabase/issues/49022
-        it.skip("3rd stage aggregation", () => {
```

Comment:
Invalid case as we've decided `Lib.ensureFilterStage` won't add a new stage when last query stage has aggregations but no breakouts.